### PR TITLE
The actual accel ramp

### PR DESCRIPTION
**Pull request by onedeadeye**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Drive.java [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
   private DriveControlState mDriveControlState;
 
-  private PeriodicIO mPeriodicIO;
```
Comment (Barrelwolf38087) [inline src/main/java/frc/robot/subsystems/Drive.java:32]: kek

**Review by maxh65000 — APPROVED** [review]

**Review by Barrelwolf38087 — APPROVED** [review]
U+24C0

File: src/main/java/frc/robot/Config.java [hunk 2]
```diff
@@ -26,6 +26,7 @@
     STORAGE__ROLLER_SPEED_FACTOR(0.5d),
     STORAGE__ROLLER_BOTTOM_SPEED_FACTOR(1.25d),
     STORAGE__ROLLER_EJECT_SPEED(1d),
+    DRIVE__ACCEL_RAMP_SPEED(1d),
```
Comment (jhinspeter) [inline src/main/java/frc/robot/Config.java]: Rename this `DRIVE__ACCEL_RAMP_TIME_SECONDS` - it's a period not a speed.

File: src/main/java/frc/robot/subsystems/Drive.java [hunk 3]
```diff
@@ -125,6 +142,31 @@ public synchronized void setOpenLoop(DriveSignal signal) {
...
+
+    // If our acceleration is positive (going away from where we were last time)
+    // Remember that right has to be flipped down below so the bracket is the other way 'round
+    mDriveLogger.log("Left: " + signal.getLeft() + " + " + mPeriodicIO.left_old);
+    mDriveLogger.log("Right: " + signal.getRight() + " + " + mPeriodicIO.right_old);
+    if ((Math.abs(signal.getLeft()) > mPeriodicIO.left_old) && (Math.abs(signal.getRight()) > mPeriodicIO.right_old)) {
+      mDriveLogger.log("setting ramp to number not zero");
+      // Tell the talons to be significantly less epic
+      mLeftMaster.configOpenloopRamp(Config.getInstance().getDouble(Key.DRIVE__ACCEL_RAMP_SPEED));
+      mRightMaster.configOpenloopRamp(Config.getInstance().getDouble(Key.DRIVE__ACCEL_RAMP_SPEED));
```
Comment (jhinspeter) [inline src/main/java/frc/robot/subsystems/Drive.java]: Why not make this a function:
```
configOpenLoopRamp(double rampValue) {
      mDriveLogger.log("setting ramp to " + rampValue);
      mLeftMaster.configOpenloopRamp(rampValue);
      mRightMaster.configOpenloopRamp(rampValue);
}

**Review by jhinspeter — DISMISSED** [review]

**Pull request merged**